app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -1,180 +1,200 @@
+                    android:layout_marginTop="10dp"
+                    android:fontFamily="@font/epilogue_semibold"
+                    android:text="@string/create_an_account"
+                    android:textColor="#9B9B9B"
+                    android:textSize="16sp"
+                    app:layout_constraintStart_toStartOf="@+id/tvSignUp"
+                    app:layout_constraintTop_toBottomOf="@+id/tvSignUp" />
+
+                <TextView
+                    android:id="@+id/tvName"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="44dp"
+                    android:text="@string/name"
+                    android:textColor="#323842"
+                    android:textSize="15sp"
+                    app:layout_constraintStart_toStartOf="@+id/tvCreateAccount"
+                    app:layout_constraintTop_toBottomOf="@+id/tvCreateAccount" />
+
+                <EditText
+                    android:id="@+id/etName"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="12dp"
+                    android:background="@drawable/custom_edittext"
+                    android:elevation="5dp"
+                    android:minWidth="269dp"
+                    android:minHeight="35dp"
+                    android:textColor="@color/black"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/tvName" />
+
+                <TextView
+                    android:id="@+id/tvEmail"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="20dp"
+                    android:text="@string/email"
+                    android:textColor="#323842"
+                    android:textSize="15sp"
+                    app:layout_constraintStart_toStartOf="@+id/tvName"
+                    app:layout_constraintTop_toBottomOf="@+id/etName" />
+
+                <EditText
+                    android:id="@+id/etEmail"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="12dp"
+                    android:background="@drawable/custom_edittext"
+                    android:elevation="5dp"
+                    android:hint="@string/hint_email"
+                    android:minWidth="269dp"
+                    android:minHeight="35dp"
+                    android:textColor="@color/black"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/tvEmail" />
```

Comment:
No need to use minHeight and minWidth. Make the width to match parent and then add sufficient horizontal margin using layout_marginHorizontal attribute.